alexandermendes/im-cli
# feature/FAB-8852 (#38)

* add-slack-integration (#12)

* Begin slack integration

* Return the pr

* add-slack-integration (#13)

* Begin slack integration

* Return the pr

* Allow dirty repos on pr

* Return PR

* Fix merge conflicts

* add-slack-integration (#16)

* Begin slack integration

* Return the pr

* Allow dirty repos on pr

* Return PR

* Fix merge conflicts

* Update Slack PR text

* add-slack-integration (#25)

* Begin slack integration

* Return the pr

* Allow dirty repos on pr

* Return PR

* Fix merge conflicts

* Update Slack PR text

* Use attatchements for Slack messages

* Abort when dirty

* add-slack-integration (#26)

* Begin slack integration

* Return the pr

* Allow dirty repos on pr

* Return PR

* Fix merge conflicts

* Update Slack PR text

* Use attatchements for Slack messages

* Abort when dirty

* Raise abort when dirty

* update-slack-message (#28)

* Add colour

* Ad newline to end of file

* Add author name and link

* Switch text for pretext

* add-slack-integration (#27)

* Begin slack integration

* Return the pr

* Allow dirty repos on pr

* Return PR

* Fix merge conflicts

* Update Slack PR text

* Use attatchements for Slack messages

* Abort when dirty

* Raise abort when dirty

* Don't repeat file extensions in slack message

* update-slack-message (#29)

* Add colour

* Ad newline to end of file

* Add author name and link

* Switch text for pretext

* Add author_icon to slace messages

* Deduplicate file extensions

* Add issue key to PR notifications

* Add missing command and move confirm

* Fix var name

* test/FAB-14293 (#30)

* Add issue key to PR notifications

* Add missing command and move confirm

* Fix var name

* Use short GitHub URL

* test/FAB-14293 (#31)

* Add issue key to PR notifications

* Add missing command and move confirm

* Fix var name

* Use short GitHub URL

* Fix var name

* Fix jira base url

* Fix issue url path

* Use button for jira link

* test/FAB-14293 (#34)

* Add issue key to PR notifications

* Add missing command and move confirm

* Fix var name

* Use short GitHub URL

* Fix var name

* Fix jira base url

* Fix issue url path

* Use button for jira link

* Fix button

* Remove unused exception

* Remove unused exception (#35)

* Fix issue URL

* feature/FAB-8852 (#36)

* Remove unused exception

* Fix issue URL

* Use GitHub open issue colour

* Update docstring
